wallet/src/de/schildbach/wallet/data/BlockchainIdentityData.kt

```diff
@@ -31,6 +31,7 @@ import org.dashevo.dpp.identity.IdentityPublicKey
 data class BlockchainIdentityData(var creationState: CreationState = CreationState.NONE,
                                   var creationStateErrorMessage: String?,
                                   var username: String?,
+                                  var userId: String?,
```

Comment:
We need to store the userId because if we are restoring a recovery phrase, we won't have the credit funding transaction until later in the sync process.